### DISCUSSION TITLE
Qt client: tts event when streaming and stop streaming

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1724] [Ideas]
### Description

  so when pressing controle s and selecting the file and pressing enter to stream, when it starts streaming, the stream option in client menu become as chekked: so i think that we can add a tts event when the file starts streaming and that stream item in client menu become as chekked, and a tts when we stop streaming and that item becomes not chekked: probably a new option in settings>text to speech tab, like stream transmission toggeled or something like that: @CoBC are you able to do that possibly? thanks in advance!

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by CoBC** [comment]
Why you want a TTS for this event? Are you not hearding your stream?

**Comment by amirmahdifard** [comment]
@CoBC whel, like other tts events like voice, video, and desktop, it is also good for havving this tts event to be noted that now you are streaming and now you stopped streaming: if it is possible for you, please add it: thank you so many!

**Comment by CoBC** [comment]
OK, but you don't answer to my question, why you want this new event, you already hear your stream start and stop, so I don't really see the advantage of a TTS event here.

**Comment by amirmahdifard** [comment]
no, sometime i don't here my stream stopps, because of my connection, and i like a tts event here, bettar than just silencely my stream stoppes: anyway, this can be optional  in settings for who don't wants it

**Comment by amirmahdifard** [comment]
so, if possible, can you add it please, since we also don't have sound event for this, but i like tts bettar than sound event

**Comment by ChrisDuffley** [comment]
@amirmahdifard @CoBC I'd suggest a "streaming stopped" event for this reason, but why something for starting streaming when clearly you can hear your audio file start streaming to the channel unless that subscription is disabled?

**Comment by amirmahdifard** [comment]
sometimes due to my connection, i cant hear my streaming for a while, and yeah, subscription is also a kace

**Comment by amirmahdifard** [comment]
for example, i can not hear that i started a  stream for even 8 secunds sometymes due to my connection, or if media volume is down we can not understand, or if the audio is silence for some tyme of it

**Comment by amirmahdifard** [comment]
@CoBC ok, now again i'm on my laptop, so i can right more bettar: so, sometymes, problems with cutting or delaying your media file due to your internet connection, and also, sometymes, there are some silence files that we can play, or, by default, in the connection tab, we disabled media file subscription, or we turned down the media file volume that we can't hear it: and since this would be optional, if anyone doesn't like it, they can personelise their teamtalk settings as how they want too: so please, if you can, work on this, thank you.

**Comment by amirmahdifard** [comment]
i'm sorry, if i knew how to open pull requests in github, instead of always asking the developers, i would've tryed fixing some other bugs too, and i would try to add my ideas my self in to the app with pull requests: but i can't open pull requests so, i'm sad that i can't do any help in my end.

**Comment by amirmahdifard** [comment]
@CoBC look: it is allmoste 24 hours after i posted this, and you asked me questions and you convurted this tikit to here, and we answered your question, and then, you didn't say anything after that, and whenever i come here and ask you to answer, people are probably gonna complane and say that you are sending so meny messages, you are pinging so much, and blaablaa: so, please read the comments that we made here, and then, if possible, shortly reply and say if you want to implement this tts in the app, or you don't want to do that depends on our comments in here: thank you.
